src/oss/confluent.sh

```diff
@@ -978,6 +979,107 @@ print_current() {
     echo "${confluent_current}"
 }
 
+check_demo_repo_uptodate() {
+    local network_status="${1}"
+    local repo="${2}"
+
+    if [ $network_status == 1 ]; then
+      cp_version=$(confluent version | awk -F':' '{print $2;}' | awk '$1 > 0 { print substr($1,1,3)}')
+      cd $repo && git fetch --tags ; git checkout $cp_version 2> /dev/null
```

Comment:
`$CONFLUENT_CURRENT` is reset every time there is `confluent start/destroy`. That is not the desired behavior with the demo-quickstarts for them to be tied to the CP start/destroy workflow.

Also, I changed the semicolon to `&&` because I want it to fail if the previous commands fail.  This doesn't directly address the pushd/popd -- what's the reasoning for using pushd/popd instead of `&&` ?

Reply:
We can revisit using `pushd/popd` in a follow up.